manifests/b/BornToBeRoot/NETworkManager/2022.2.22/BornToBeRoot.NETworkManager.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: BornToBeRoot.NETworkManager
+PackageVersion: 2022.2.22
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2022-02-22
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/BornToBeRoot/NETworkManager/releases/download/2022.2.22/NETworkManager_2022.2.22.0_Setup.exe
```

Comment:
URL changed to https://github.com/BornToBeRoot/NETworkManager/releases/download/2022.2.22.0/NETworkManager_2022.2.22.0_Setup.exe